targets/TARGET_STM/TARGET_STM32F7/device/stm32f7xx_hal_pcd.c

```diff
@@ -148,8 +149,9 @@ HAL_StatusTypeDef HAL_PCD_Init(PCD_HandleTypeDef *hpcd)
   {
     /* Allocate lock resource and initialize it */
     hpcd->Lock = HAL_UNLOCKED;
-	for (i = 0; i < hpcd->Init.dev_endpoints ; i++)
-	hpcd->EPLock[i].Lock = HAL_UNLOCKED;
+    for (i = 0; i < hpcd->Init.dev_endpoints ; i++) { // MBED PATCH
```

Comment:
I didn't see this change in the other PRs. Is this just cosmetic?

Reply:
Yes just cosmetic